### DISCUSSION TITLE
Migrating to Uno Material V2

### DISCUSSION THREAD

**Discussion by Arieldelossantos** [unoplatform/uno#11909] [Q&A] [answered]
While migrating to Uno Material V2 i've noticed that the buttons are now Rounded specifically the "FilledButtonStyle" which is the default style... is there a global way to set this to not rounded buttons, other than creating a local style to override my buttons??

another Question is that the Documentation has a table with [the matrix of new styles](https://platform.uno/docs/articles/external/uno.themes/doc/material-controls-styles.html), however, if you go to the [Styles/Control/V2 code repo](https://github.com/unoplatform/Uno.Themes/tree/master/src/library/Uno.Material/Styles/Controls/v2) to check the V2 styles you'll notice that most of them(almost all) have still the "Material" prefix... which one is the correct one???

**Comment by kazo0** [comment, marked as answer]
Hi @Arieldelossantos!

Yes, the Uno.Material library as of v2 now defaults to the [Material Design 3](https://m3.material.io/) guidance for things like [Buttons](https://m3.material.io/components/all-buttons), etc. If you would like to continue using the [Material Design 2](https://m2.material.io/components) styles, you should be able to follow the steps [here](https://platform.uno/docs/articles/external/uno.themes/doc/material-migration.html#continue-using-v1-styles)

Otherwise, you would need to provide your own style overrides and change the CornerRadius for your Buttons in your application.

As for the style keys, you can actually reference the styles with or without the "Material" prefix, all "Material"-prefixed styles are aliased to keys that do not include the "Material" prefix [here](https://github.com/unoplatform/Uno.Themes/blob/54007e0fc96c35b768f476206b65e561eee4497d/src/library/Uno.Material/Styles/Controls/v2/_Resources.xaml#L40).
